person.rb

```diff
@@ -1,7 +1,8 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
```

Comment:
- [ ] Dear @MNisarAli, I notice you have  `person.rentals << self` while you initialize rentals, But it's mandatory to add a method to add new rentals to the person. Please consider adding a `add_rental(book, date)` method to add a rental to the current person. [More here about both-way relationship](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)